app/utils/time.py

```diff
@@ -11,7 +11,7 @@ def get_current_financial_year():
     now = datetime.now(preferred_tz)
     current_month = int(now.strftime("%-m"))
     current_year = int(now.strftime("%Y"))
-    return current_year if current_month > 9 else current_year - 1
+    return current_year if current_month < 10 else current_year + 1
```

Comment:
Just checking my understanding here:  this is the switch to basing the fiscal year starting in October, so if we're before October we're in the current calendar year, and if we're after then technically we're in the next calendar year, right?

Reply:
Yeah, exactly!

Reply:
Thanks!